share/completions/tox.fish

```diff
@@ -0,0 +1,35 @@
+function __toxenvs
```

Comment:
should call this `__fish_toxenvs` or `__fish_tox_complete_envs` for consistency with similar code